README_BACKEND.md

```diff
@@ -0,0 +1,13 @@
+## How to build a Docker container
```

Comment:
Hi! sorry for sneaking in. I am a frontend mentor in the H2G program.

Can I ask you to make a more detailed step by step readme about how to start the application?
* Assume that the user doesn't know anything about docker/java/nvm/etc
* Assume nothing is installed in the user's machine
* Please provide info how to use it, for example give an example js code calling the endpoints. Something like that.

Thank in advance!

Reply:
After making the requirement more accurate by @hegedus-adam.
Docker is still requirement, java/mvn hide behind docker and CI pipeline.